### PR TITLE
use common ci mgmt definition

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
Fix https://github.com/Azure/azure-sdk-for-java/issues/6385

### CODE REVIEW

**Comment by weidongxu-microsoft** [comment]
@mitchdenny 

Let me know whether this is OK for you.

CI fails on client/data probably not related to this PR. I only modifies mgmt CI.

**Comment by weidongxu-microsoft** [comment]
BTW, mgmt CI works pretty good. You can delete the Azure.azure-sdk-for-java-mgmt CI if you wishes.

File: sdk/vmwarecloudsimple/ci.mgmt.yml [hunk 1]
```diff
@@ -13,9 +13,7 @@ trigger:
       - release/*
   paths:
     include:
-      - sdk/vmwarecloudsimple/
-    exclude:
-      - sdk/vmwarecloudsimple/microsoft-azure
+      - sdk/vmwarecloudsimple/mgmt-
```
Comment (weshaggard) [inline sdk/vmwarecloudsimple/ci.mgmt.yml]: Are you sure this path prefix works? I don't believe DevOps allows such syntax so I'd suggest you validate it.

Comment (mitchdenny) [reply sdk/vmwarecloudsimple/ci.mgmt.yml]: The matching logic for pipelines is effectively a starts-with check. So if your folkders start with ```sdk/vmwareclouresimple/mgmt-``` it should match (and in this case exclude) them.

Comment (mitchdenny) [reply sdk/vmwarecloudsimple/ci.mgmt.yml]: ... you just can't/don't need the wildcard symbols like ```*``` etc.

**Comment by mitchdenny** [comment]
This looks good. I've disabled the old management pipeline and moved it into the ```\java\legacy``` folder in Azure DevOps.

File: sdk/vmwarecloudsimple/ci.mgmt.yml (hunk 1 above)
Comment (weidongxu-microsoft) [reply sdk/vmwarecloudsimple/ci.mgmt.yml]: @weshaggard @mitchdenny 

I've tried part of path in my own repo. It indeed does not work.

I set the "include" to "src/ma", not work, "src/main", works.

I will revert it back to "include" + "exclude" setting. Even if "exclude" part not working correctly, it still be better compared to "include" part not working.

**Pull request merged**